pyproject.toml

```diff
@@ -1,2 +1,30 @@
-[tool.black]
-extend-exclude = '^/examples/python/pytket-qujax_heisenberg_vqe.py'
+[tool.poetry]
+name = "pytket-docs"
+package-mode = false
+version = "0.1.0"
+description = ""
+authors = ["CalMacCQ <93673602+CalMacCQ@users.noreply.github.com>"]
+readme = "README.md"
+
+[tool.poetry.dependencies]
+python = "^3.10"
```

Comment:
Do we have dependabot set up to check for for updates?

Reply:
Not currently I think. I can look into setting that up?

Reply:
Would you prefer to just check for pytket and extensions updates? Or the docs dependencies as well?

Reply:
Maybe this dependabot config is already valid for `pyproject.toml`?

https://github.com/CQCL/pytket-docs/blob/main/.github/dependabot.yml

Reply:
Yes, that should probably cover everything.